administrator/language/en-GB/en-GB.com_joomlaupdate.ini

```
@@ -79,6 +79,7 @@ COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSION_WARNING_UNKNOWN="Unkown error"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSION_SERVER_ERROR="Update Server error"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS="Extensions Pre-Update Check"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_NONE="No extensions installed."
+COM_JOOMLAUPDATE_VIEW_DEFAULT_NON_CORE_BACKEND_TEMPLATE_USED_NOTICE="We detected that you are not using a core admin template. You might find the upgrade process smoother if you <a href="_QQ_"index.php?option=com_templates&view=styles&client_id=1"_QQ_">switch</a> to use the Isis template."
```

To my knowledge `_QQ_` should not be used anymore.
I was told that `\"` is the correct way nowadays.

---

this is a j3 string not j4

---

J3 also supports `\"` and already uses that on a lot of places, which makes it compatible with J4 as well.

---

```suggestion
COM_JOOMLAUPDATE_VIEW_DEFAULT_NON_CORE_BACKEND_TEMPLATE_USED_NOTICE="We detected that you are not using a core admin template. You might find the upgrade process smoother if you <a href='index.php?option=com_templates&view=styles&client_id=1'>switch</a> to use the Isis template."
```

That should do the trick too right?

---

I would think so comparing it to other strings I have seen.